### ISSUE TITLE
New Device: Have you already or do you plan to add Alperen-Ozturk's "Switch" device?

### ISSUE THREAD

**Issue by aketterson** [iRayanKhan/homebridge-tuya#190]
I have a TreatLife wall switch that I am using in HomeBridge with Alperen-Ozturk's "Switch" implementation of Tuya-LAN [https://github.com/Alperen-Ozturk/homebridge-tuya-switch]  Do you plan to incorporate this into your next release or is there another Device Type that works just as well?  I use the Switch to also control non-Tuya devices.

**Comment by pst-nn** [comment]
I've looked into the code and it's already merged, the configuration is identical to "Alperen-Ozturk's".

However, there's unexpected behavior. I have a 3 gang switch that controls multiple rooms. When using "homebridge-tuya-web", these 3 switches display separately as 3 devices, which allows me to assign them to different rooms. But which this "homebridge-tuya" plug-in, appears as 1 device with 3 switches, as the result, I can assign them to only 1 room, even if I chose to "Show as Separate Tiles", they always follow each other to the same room. Is there any workaround on this?
